lib/explorer/polars_backend/shared.ex

```diff
@@ -133,7 +133,13 @@ defmodule Explorer.PolarsBackend.Shared do
   end
 
   def from_list(list, {:struct, fields}, name) when is_list(list) do
-    columns = Table.to_columns(list)
+    columns =
+      list
+      |> Enum.map(fn
+        nil -> Enum.map(fields, fn {k, _} -> {k, nil} end)
+        x -> x
+      end)
+      |> Table.to_columns()
 
```

Comment:
Need input -  Should check if the first non nil entry has atom keys or string keys, 

As `k` is always a string - it will fail if input has atom keys. 
```elixir
Series.from_list([%{a: 1}, nil])


** (RuntimeError) key-value records must have columns in the same order, expected :a, but got "a"
    (table 0.1.2) lib/table/reader/enumerable.ex:100: Table.Reader.Enumerable.keyval_values/2

```